speech/speech.bas

```diff
@@ -55,7 +55,7 @@ PRINT Intro$
 DO
         INPUT "Please enter something for me to speak, or Q to exit";Answer$
 
-Test:   IF Answer$="Q" OR Answer$="q" THEN END
+Test:   IF Answer$="Q" OR Answer$="q" THEN we.quit% = 1 : we.end_program

```

Comment:
See previous comment about using we.is_quit_key%()

Reply:
Done - changed to use we.is_quit_key%()